src/main/modOrganizer.ts

```diff
@@ -149,13 +149,51 @@ export async function launchGame() {
   }
 }
 
-export const getProfiles = async (): Promise<string[]> =>
-  (
+export async function getProfiles(): Promise<string[]> {
+  // get mapped profile names
+  const mappedNames = JSON.parse(
+    await fs.promises.readFile(
+      `${userPreferences.get(
+        USER_PREFERENCE_KEYS.MOD_DIRECTORY
+      )}/profiles/namesMO2.json`,
+      "utf-8"
+    )
+  );
+
+  // get unmapped profile names
+  const unmappedNames = (
     await fs.promises.readdir(
       `${userPreferences.get(USER_PREFERENCE_KEYS.MOD_DIRECTORY)}/profiles`,
       { withFileTypes: true }
     )
   )
     .filter((dirent) => dirent.isDirectory())
     .map((dirent) => dirent.name)
-    .filter(isNotJunk);
+    .filter(isNotJunk)
+    .map((dirent) => ({ real: dirent, friendly: dirent }))
+    .filter((dirent) => {
+      return !mappedNames.find(
+        (preset: FriendlyDirectoryMap) => preset.real == dirent.real
+      );
+    });
```

Comment:
I would use a different parameter name to make it clearer.
I used `dirent` above because it was a [FileSystemEntity](https://developer.mozilla.org/en-US/docs/Web/API/FileSystemEntry) from passing `{ withFileTypes: true }` (so we could remove none directories).

However, after `.map((dirent) => dirent.name)` the is just an array of strings.
Always use `===` instead of `==` too.

```suggestion
    .map((preset) => ({ real: preset, friendly: preset}));
    .filter((unmappedPreset) => {
      return !mappedNames.find(
        (mappedPreset: FriendlyDirectoryMap) => mappedPreset.real === unmappedPreset
      );
    });
```